numpythia/src/_libnumpythia.pyx

```diff
@@ -62,6 +63,16 @@ PARENTS = HepMC.PARENTS
 CHILDREN = HepMC.CHILDREN
 SIBLINGS = HepMC.PRODUCTION_SIBLINGS
 
+cdef class sphericity:
```

Comment:
I would keep this "normally" named, `Sphericity`, and instead renamed the `Sphericity` when you import it, say to `cSphericity`. (Sadely, the current code is bad about this, but ideally a wrapper should as similar to the original code as possible when making "arbitrary" choices, like names).